### ISSUE TITLE
Option to choose front or back camera during the camera permission

### ISSUE THREAD

**Issue by KrasnayaPloshchad** [mozilla-mobile/fenix#15993]
[comment]: # (Please do your best to search for duplicate issues before filing a new issue so we can keep our issue board clean)
[comment]: # (Every issue should have exactly one feature request described in it. Please do not file feedback list tickets as it is difficult to parse them and address their individual points)
[comment]: # (Feature Requests are better when they’re open-ended instead of demanding a specific solution e.g: “I want an easier way to do X” instead of “add Y”)
[comment]: # (Read https://github.com/mozilla-mobile/fenix#i-want-to-file-an-issue for more information)

### What is the user problem or growth opportunity you want to see solved?
After #15421 the camera is now default to front one during the permission, but there are some requests to create a drop down menu to select which camera to be used during the permission, that is what you’ve made before Fenix.
### How do you know that this problem exists today? Why is this important?
There are some sites requiring camera for reading QR codes, but doesn’t give permission to choose which camera to use, without this option it’s inconvenient to use them sometimes, such as https://qrcodescan.in/, https://www.qrstuff.com/scan, https://www.the-qrcode-generator.com/scan, https://8w.8thwall.app/scan/ With such sites it’s possible to reproduce this issue.
### Who will benefit from it?
Anyone who needs to switch the camera in Firefox, and who is using device with more than one camera.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-8269)

**Comment by liuche** [comment]
Opera has this functionality but no other mobile browsers.

**Comment by KrasnayaPloshchad** [comment]
Fennec also have this functionality.

**Comment by SeburoSUMO** [comment]
User feedback: https://support.mozilla.org/en-US/questions/1344161

**Comment by skbeh** [comment]
@liuche The feature is very important for online meeting. Anyone who use camera benefits from it.

**Comment by jonalmeida** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1813231

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
